### PR TITLE
setup: mono-repo

### PR DESCRIPTION
**Pull request by nihalanisumit**

### CODE REVIEW

**Review by karansapolia — APPROVED** [review]
LGTM. The codeclimate check can be bypassed on this one. We are looking into why is it waiting indefinitely.

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+node_modules
+package-lock.json
+.vscode
```
Comment (rishabh1403) [inline .gitignore]: new line :)

**Review by karansapolia — APPROVED** [review]
LGTM. Comments same as before.

File: client/src/index.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+import './index.css';
+import App from './App';
+import * as serviceWorker from './serviceWorker';
+
+ReactDOM.render(<App />, document.getElementById('root'));
+
+// If you want your app to work offline and load faster, you can change
+// unregister() to register() below. Note this comes with some pitfalls.
+// Learn more about service workers: https://bit.ly/CRA-PWA
+serviceWorker.unregister();
```
Comment (rishabh1403) [inline client/src/index.js]: are you using service worker ?

File: client/src/serviceWorker.js [hunk 3]
```diff
@@ -0,0 +1,135 @@
+// This optional code is used to register a service worker.
+// register() is not called by default.
+
+// This lets the app load faster on subsequent visits in production, and gives
+// it offline capabilities. However, it also means that developers (and users)
```
Comment (rishabh1403) [inline client/src/serviceWorker.js]: is this file being used?

File: server/index.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
+const express = require("express");
+const app = express();
+const port = 3001;
```
Comment (rishabh1403) [inline server/index.js:3]: environment variable :)

**Review by rishabh1403 — APPROVED** [review]
Minor Changes then LGTM

**Review by piyushagwl007 — APPROVED** [review]
LGTM. I am thinking of skipping the code climate check, for now, as it is not responding.

File: client/src/index.js (hunk 2 above)
Comment (nihalanisumit) [reply client/src/index.js]: No, but maybe in future. I will re-add it if required. removed for now.

File: server/index.js (hunk 4 above)
Comment (nihalanisumit) [reply server/index.js:3]: @piyushagwl007 has taken care of this.

Comment (piyushagwl007) [reply server/index.js:3]: Yes. I will be taking care of this in my subsequent PR.

**Pull request merged**